plugins/curl/README.md

```diff
@@ -0,0 +1,12 @@
+# Curl aliases plugin
+
+## Functions explained
+
+ * curlj - pretty json (require Python 2.6)
+
+## Aliases explained
+
+ * curlh - show header only
+ * curls - download filename
+ * curlt - show response time in milliseconds
+ * curlrc - show http code
```

Comment:
I think it would be better if you mentioned ONLY show http code and discard the rest. People might be expecting otherwise. Otherwise, LGTM!